Add wattpad-scraper

shhossain opened the pull request:
## Wattapad Scraper
If you have any problems let me know.

shhossain committed:
Adding my new project

ritwik12 commented on _data/projects/wattpad-scraper.yml:
```diff
@@ -0,0 +1,12 @@
+name: Your Project Name Here
```
@shhossain This should be `wattpad-scraper`

shhossain replied:
name changed

ritwik12 commented on _data/projects/wattpad-scraper.yml:
```diff
@@ -0,0 +1,12 @@
+  - test
+upforgrabs:
+  name: help wanted
+  link: https://github.com/shhossain/wattpad-scraper/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```
@shhossain Please change this to https://github.com/shhossain/wattpad-scraper/labels/help%20wanted

shhossain replied:
link fixed

shhossain committed:
Merge branch 'up-for-grabs:gh-pages' into patch-1

shhossain committed:
fixed name and link

name added
link fixed to https://github.com/shhossain/wattpad-scraper/labels/help%20wanted
